cilium-certgen/TAG

```diff
@@ -1 +1 @@
-0.1.8.1
+0.1.9.1
```

Comment:
Cilium v1.13.6 still uses certgen 0.1.8 by default. I think we should use v1.13.7 with certgen v0.1.9
https://github.com/cilium/cilium/blob/1.13.6/install/kubernetes/cilium/values.yaml#L843